jobserver/views/workspaces.py

```diff
@@ -183,7 +182,7 @@ def dispatch(self, request, *args, **kwargs):
             )
 
         org = self.request.user.orgs.first()
-        gh_org = first(org.github_orgs)
+        gh_org = next((o for o in org.github_orgs if o), None)
```

Comment:
thought: This feels OK to leave as is, with the default set. We explicitly check for that default in the next line.

Reply:
I've simplified this code.